src/network/network-er-x-sfp.md

```diff
@@ -0,0 +1,332 @@
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 start 100.64.12.127 stop 100.64.12.254
+
+set service dhcp-server shared-network-name 100.64.13.0 authoritative enable
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 default-router 100.64.13.1
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 dns-server 1.1.1.1
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 start 100.64.13.127 stop 100.64.13.254
+```
+
+## L2TP Tunnel
+
+L2TP Tunnels is used to connect back to the an [exit node](exit-node.md).  
+
+All commands are prefixed with `set interfaces l2tpv3 l2tpeth0` where `l2tpeth0` is a numbered interface starting from 0.
+
+Between the EXIT NODE configuration and the CLIENT node the following values are filled 
+
+```
+- destination-port <> source-port
+- session-id <> peer-session-id
+- tunnel-id <> peer-tunnel-id
+- remote-ip <> local-ip
+```
+
+`local-ip` MUST be an IP address defined on the router. This means if the device is behind a NAT it will be a local IP not public IP. Also, if using NAT sometimes source-port needs to be added to the port forward.
+
+Configure each port with a unique IPv4 and IPv6 IP address. Use a `/30` subnet for IPv4 and `/126` for IPv6. Check with the exit node you are using to confirm to the numbering selected.
+
+### Example
+
+```
+set interfaces l2tpv3 l2tpeth0 description "To Exit Node NJ"
+set interfaces l2tpv3 l2tpeth0 destination-port 5000
+set interfaces l2tpv3 l2tpeth0 local-ip 192.168.40.55
+set interfaces l2tpv3 l2tpeth0 mtu 1412
+set interfaces l2tpv3 l2tpeth0 peer-session-id 1000
+set interfaces l2tpv3 l2tpeth0 peer-tunnel-id 3000
+set interfaces l2tpv3 l2tpeth0 remote-ip 199.195.250.209
+set interfaces l2tpv3 l2tpeth0 session-id 2000
+set interfaces l2tpv3 l2tpeth0 source-port 6000
+set interfaces l2tpv3 l2tpeth0 tunnel-id 4000
+
+set interfaces l2tpv3 l2tpeth0 address 100.127.2.2/30
+set interfaces l2tpv3 l2tpeth0 address fd74:6f6d:7368:7f02::2/126
+```
+
+## Babeld
+
+Babeld does not come standard on ER-X.  Use the deb package located at  https://github.com/darkdrgn2k/RouterX-Babeld-Package  and install the deb using `dpkg -i`  on the router. Review the instructions in the repo to allow package to survive firmware upgrades.
+
+All commands below are to be prefixed with `set protocols babeld`
+
+`denydefault` and `denydefaultlocal` add a "deny all" rules to the end of the configuration. This prevents unwanted IPs to be announced such as the local IP address of the gateway network in case of a gateway.
```

Comment:
```suggestion
`denydefault` and `denydefaultlocal` add a "deny all" rules to the end of the configuration. This prevents unwanted IPs from being announced such as the local IP address of the gateway network in case of a gateway.
```